pkg/controller/account/iam.go

```diff
@@ -434,10 +436,14 @@ func deleteIAMUser(reqLogger logr.Logger, awsClient awsclient.Client, user *iam.
 	return nil
 }
 
+var (
+	listIAMUsers = awsclient.ListIAMUsers
+)
```

Comment:
Just curious, what is the advantage of it this way vs `var listIAMUsers = awsclient.ListIAMUsers` for a single variable?

Reply:
Apologies, I should have left a comment above explaining my reasoning for this - I'll add it in the morning. 
I'm doing this to monkey patch the function `awsclient.ListIAMUsers` being called with my own defined 'mocked' func found [here](https://github.com/openshift/aws-account-operator/pull/624/files#diff-2d6c1df4c89463ea67c8a31c387b6ae3c450fe8c265fd99f446b73d3236a309eR579). 

The way I would normally mock out these func calls is by using some form of `mockedawsclient.EXPECT().funcCall` - unfortunately this wasn't cleanly available to me here. The [awsclient.ListIAMUsers](https://github.com/openshift/aws-account-operator/blob/master/pkg/awsclient/iam.go#L44)  isn't defined in the client interface, therefore it isn't mocked out. What is mocked out though is the function which ListIAMUsers calls, [ListUsersPages](https://github.com/openshift/aws-account-operator/blob/master/pkg/awsclient/iam.go#L49-L53). `ListUsersPages`'s second parameter is a function pointer which modifies a locally defined slice - meaning I'd need to do a major refactor just to mock this function out differently.